src/jobs/push-to-app-catalog.yaml

```diff
@@ -13,6 +13,14 @@ parameters:
   chart:
     description: "Name of the chart inside helm directory to push to the App Catalog."
     type: "string"
+  lenient-lint:
+    description: "Use more lenient lint rules"
+    type: "boolean"
+    default: false
+  disable-lint:
+    description: "Disable linting alogether"
+    type: "boolean"
+    default: false
```

Comment:
Nitpicky but we usual use alpha order for params unless there is a good reason not to. WDYT?

Reply:
I agree, ordering is a small thing but cognitively helps a lot. Fixed in b0237c3